README.md

```diff
@@ -94,13 +103,25 @@ To begin benchmarking your first embodied agent with NETT, please be aware:
 
 ### Running a NETT
 
-1. **Download or Create the Unity Executable**
+1. **Download the Unity Executable**
+
+   You can download and unzip pre-made Unity executables from [here](https://origins-of-intelligence.com/environments/). An executable is required to run the virtual experiments.
 
-   Download and unzip a pre-made Unity executable from [here](https://origins-of-intelligence.com/environments/). The executable is required to run the virtual environment.
+   If you are running this pipeline on a server and you need to copy the executable folder from your local machine to the server, use the following:
+
+   ```bash
+   scp -r \local\path\to\executable\folder  \server\parth\to\directory\for\executable\folder\
+   ```
+   *(Note: if you are a member of our lab using one of our lab servers, the server path will be: your_username@server_name.luddy.indiana.edu:/home/your_username)*
```

Comment:
Should we be putting our personal server information on a public repo? I think it might make more sense to hold this in internal documentation (either on our Google Drive or in the buildingamind/docs repo)

Reply:
Hmm... let's talk about this together at the Wednesday meeting. It's not clear to me that this is sensitive information. The only information that's getting provided is the luddy.indiana.edu part and the fact that there's a home directory, but both of those seem pretty obvious. I really like the idea of having one place to go to for running the pipeline instead of a bunch of different pieces of documentation. But if I'm missing how this is sensitive, then we can move it. Perhaps we can get some opinions from the group on how sensitive this info is?